content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -377,70 +366,49 @@ Once the connection is established, Odoo can be synchronized with CodaBox.
+    Odoo connection on CodaBox's side (i.e. on your myCodaBox portal). Then, you will be able to
+    revoke the connection on Odoo's side and create a new one.
+
+It seems that the company or fiduciary VAT number you provided is not valid. Please check your configuration.
+  Either the Company VAT of the Accounting Firm VAT is not in a valid Belgian format.
 
 It seems that the fiduciary VAT number you provided does not exist in CodaBox. Please check your configuration.
-  The Accounting Firm VAT number you provided is not registered in CodaBox.
-
-Error while connecting to the IAP server. Please contact Odoo support.
-  It seems that Odoo's IAP server is down or unreachable. Please contact
-  Odoo support.
-
-It seems that your CodaBox connection is not valid anymore. Please check your configuration.
-  It seems that you have either revoked the access of Odoo to your CodaBox account, or that
-  you have not completed the configuration process. In this case, you must click on the
-  :guilabel:`Revoke connection` button in the settings, and follow again the steps in
-  the :ref:`first connection <belgium/codabox-configuration-connection>` section.
-
-A previous CodaBox registration already exists. Paste the access token from a previous connection here. Odoo will generate a new one, exclusive to this database.
-  It seems that you have already your Accounting Firm with Odoo, but that the Access Token
-  is either not filled or wrong. You should follow the steps in the
-  :ref:`following connection <belgium/codabox-configuration-connection>` tab section.
-
-  If you have lost your Access Token, you have to go on the CodaBox platform,
-  and revoke the access of Odoo to your CodaBox account. Then, you will have to follow
-  the steps in the :ref:`first connection <belgium/codabox-configuration-connection>` section.
-
-The provided access token is not valid for this fiduciary. Please check your configuration. If you have lost your access token, please contact Odoo support.
-  Same as above.
-
-It seems that no connection linked to your database/VAT number exists. Please check your configuration.
-  You may obtain this error when revoking or trying to fetch files from CodaBox if
-  the pair (Accounting Firm VAT, Company VAT) is not registered yet. This may happen if
-  you have changed the company VAT after the connection was established for instance.
-  For security reasons, you have to recreate a connection for this company VAT by following the
-  steps in the :ref:`first connection <belgium/codabox-configuration-connection>` section.
-
-No files were found. Please check your configuration.
-  This may occur if the company VAT has changed after the connection was established.
-  In that case, you must click on the :guilabel:`Revoke connection` button in the settings,
-  and recreate the connection.
+  The Fiduciary VAT number you provided is not registered in CodaBox. You may not have a valid
```

Comment:
The Accounting Firm VAT number you provided is not registered in CodaBox. You may not have a valid